cicecore/cicedyn/general/ice_init.F90

```diff
@@ -2692,14 +2692,16 @@ subroutine input_data
           kmt_type  /=  'channel_onenorth' .and. &
           kmt_type  /=  'wall'    .and. &
           kmt_type  /=  'default' .and. &
-          kmt_type  /=  'boxislands') then
+          kmt_type  /=  'boxislands'.and. &
+          kmt_type  /=  'none' ) then
          if (my_task == master_task) write(nu_diag,*) subname//' ERROR: unknown kmt_type=',trim(kmt_type)
          abort_list = trim(abort_list)//":27"
       endif
 
       if (grid_type  /=  'column'      .and. &
           grid_type  /=  'rectangular' .and. &
-          kmt_type   /=  'file') then
+          kmt_type   /=  'file' .and. &
+          kmt_type   /= 'none') then
```

Comment:
Also, the supergrid does not have depth. There is a separate bathymetry file. "KMT" is the old style pop depth file which just lists the lowest depth level which is water. We use KMT to derive the mask, but also the basal stress formulation will use the actual depth.

Reply:
I was just using kmt_type of 'none' for testing, for now in runs we would still use kmt_type = file

In some ways, it might be better to rename kmt_type to mask_type with options of `kmt_file`, `mask_file` and in the future `bathymetry_file` ?